examples/cross_build/emscripten/bindings/conanfile.py

```diff
@@ -0,0 +1,38 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+
+
+class WasmExampleRecipe(ConanFile):
+    name = "wasm-example"
+    version = "1.0"
+    package_type = "application"
+    settings = "os", "compiler", "build_type", "arch"
+
+    def layout(self):
+        cmake_layout(self)
+
+    def requirements(self):
+        self.requires("eigen/3.4.0")
+        self.requires("zlib/1.3.1")
+        self.requires("fmt/11.1.4")
+
+    def generate(self):
+        deps = CMakeDeps(self)
+        deps.generate()
+        tc = CMakeToolchain(self)
+
+        # HEAPxx values need to be exported explicitly since Emscripten 4.0.7  
+        # https://github.com/emscripten-core/emscripten/blob/main/ChangeLog.md#407---041525
+        tc.extra_exelinkflags.append(
```

Comment:
this is not best practice, as it is hardcoded in recipe for all profiles.
Either this recipe is exclusively for webassembly, or maybe define it elsewhere? I understand that the ``main.cpp`` is only for emscripten right now, so maybe a ``validate()`` that raises if not ``emcc``?